### PR TITLE
[NLU-3867] Arabic Currency Support

### PR DESCRIPTION
**Pull request by samhickey25**

### CODE REVIEW

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/__init__.py [hunk 1]
```diff
@@ -11,3 +8,5 @@
 from .portuguese_numeric_with_unit import PortugueseNumericWithUnit
 from .spanish_numeric_with_unit import SpanishNumericWithUnit
 from .japanese_numeric_with_unit import JapaneseNumericWithUnit
+from .catalan_numeric_with_unit import CatalanNumericWithUnit
```
Comment (colm-dillon) [inline Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/__init__.py:11]: 🤔 - seems not to have impacted Catalan amountOfMoney

**Review by colm-dillon — APPROVED** [review]

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/__init__.py (hunk 1 above)
Comment (samhickey25) [reply Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/__init__.py:11]: it's probably not necessary but I put it in for consistency

**Review by andrew-gradinari — APPROVED** [review]

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+
+
+# pylint: disable=line-too-long
+
+
+class ArabicNumericWithUnit:
+    CurrencySuffixList = dict([("Abkhazian apsar", "أبخازية أبسار"),
+                               ("Afghan afghani", "أفغاني أفغاني|؋|الأفغاني|أفغاني"),
+                               ("Pul", "بول"),
+                               ("Euro", "يورو|اليورو|€|يورو"),
```
Comment (kevinwalshgen) [inline Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py]: I see some, like Australian Dollar have AUD, US Dollar have USD specified as well, so should this include `EUR`?

**Review by kevinwalshgen — APPROVED** [review]

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py (hunk 2 above)
Comment (samhickey25) [reply Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py]: ah yeah, no harm

**Pull request merged**